pkg/domain/entities/types/auth.go

```diff
@@ -1,4 +1,4 @@
-package types // import "github.com/docker/docker/api/types"
```

Comment:
Why did you change these?

Reply:
that's the main thing that breaks build with `GO111MODULE=off`. Please let me know if there's a better option to handle that.